common/src/main/java/com/wynntils/models/containers/ContainerModel.java

```diff
@@ -50,12 +50,14 @@ public final class ContainerModel extends Model {
     private static final Pair<Integer, Integer> CONTENT_BOOK_PREVIOUS_NEXT_SLOTS = new Pair<>(65, 69);
     private static final Pair<Integer, Integer> GUILD_BANK_PREVIOUS_NEXT_SLOTS = new Pair<>(9, 27);
     private static final Pair<Integer, Integer> GUILD_MEMBER_LIST_PREVIOUS_NEXT_SLOTS = new Pair<>(10, 28);
+    private static final Pair<Integer, Integer> JUKEBOX_PREVIOUS_NEXT_SLOTS = new Pair<>(17, 8);
```

Comment:
... wait a sec. Don't you need to reference this as well? You need to add the jukebox screen as scrollable, somehow?

Reply:
@DonkeyBlaster 

I did not see that you had auto merge turned on, so this comment arrived too late.

Reply:
Hmm.... I swear I did the normal implementation and everything and even played with it in-game. Guess not. I'll follow up on this PR with the search and fixes in a bit.